routes/message.routes.js

```diff
@@ -5,16 +5,39 @@ router.get("/message", (req, res) => {
   res.json("Hello world");
 });
 
-
 router.post("/messages", (req, res) => {
   const { destination, message } = req.body;
 
-  const body = message
+  const body = message;
 
   MessageAppService.sendMessage(destination, body)
     .then((response) => res.json(response.data))
-    .catch((err) => res.json(err));
+    .catch((err) => {
+      if (
+        !err.config.data.includes("destination") &&
```

Comment:
I'm checking config because I'm directly checking the errors instead of preventing them, so I can tell the client which error is occurring and how to fix it.